crypto/src/client.rs

```diff
@@ -98,6 +100,18 @@ fn identity_key(credentials: &CredentialBundle) -> Result<Vec<u8>, MlsError> {
 }
 
 impl Client {
+    /// Initialize a new client.
+    ///
+    /// # Arguments
+    /// * `id` - id of the client
+    /// * `certificate_bundle` - an optional x509 certificate
+    /// * `backend` - the KeyStore to read identities from
+    ///
+    /// # Errors
```

Comment:
I'm not sure documenting errors like this brings value. We will end up most of the time with 3 kind of errors: keystore, openmls & corecrypto.  
Instead, I think documenting the errors conditions would bring more value e.g.
```rust
/// # Errors
/// Fails if:
/// - [condition-1]
/// - [condition-2] etc..
```